src/main/java/frc/robot/systems/IntakeFSMSystem.java

```diff
@@ -46,6 +46,7 @@ public enum IntakeFSMState {
 	private Slot0Configs slot0Configs = talonFXConfigs.Slot0;
 	private MotionMagicConfigs motionMagicConfigs = talonFXConfigs.MotionMagic;
 	private StatusCode statusCode = StatusCode.StatusCodeNotInitialized;
+	private TeleopInput tInput;
```

Comment:
There should only ever be one TeleopInput at Robot.java level. No need to create your own